supporting_content/attributes/discount_handling.md

```diff
@@ -0,0 +1,117 @@
+  - AWS partner tiers - https://aws.amazon.com/partners/services-tiers/
+
+- GCP
+  - Committed use discounts - https://cloud.google.com/compute/docs/instances/committed-use-discounts-overview
+  - Resource-based CUDs - https://cloud.google.com/compute/docs/instances/signing-up-committed-use-discounts
+  - Spend-based CUDs - https://cloud.google.com/docs/cuds-spend-based
+  - Sustained use discounts - https://cloud.google.com/compute/docs/sustained-use-discounts
+  - Credit types - https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/standard-usage#standard-usage-cost-data-schema
+  - GCP Partner Advantage - https://cloud.google.com/partners/become-a-partner
+
+- Microsoft
+  - Reservations - https://learn.microsoft.com/en-us/azure/cost-management-billing/reservations/save-compute-costs-reservations
+  - Savings Plans - https://learn.microsoft.com/en-us/azure/cost-management-billing/savings-plan/savings-plan-compute-overview
+  - Azure Hybrid Benefit - https://azure.microsoft.com/en-us/pricing/hybrid-benefit/#overview
+  - Azure Dev/Test Pricing - https://azure.microsoft.com/en-us/pricing/offers/dev-test/
+  - Microsoft Partnership - https://partner.microsoft.com/en-us/partnership/compare-programs
+
+# Discussion / Scratch space
+
+- AWS discount types include:
+  - DiscountedUsage - discounts covered by Reserved Instances
+  - SavingsPlanCoveredUsage - discounts covered by SavingsPlans
+  - EdpDiscount - Enterprise Discount Program
+  - SppDiscount - Solutions Provider Program
+  - RiVolumeDiscount
+  - BundledDiscount - discounted usage on a specific product/service based on the usage of another product/service.
+
+- GCP discount types include:
+  - DISCOUNT - credits earned after a contractual spending threshold is reached.
+  - COMMITTED_USAGE_DISCOUNT - resource-based commitment discounts.
+  - COMMITTED_USAGE_DISCOUNT_DOLLAR_BASE - Spend-based commitment discounts.
+  - SUSTAINED_USAGE_DISCOUNT - automatic discount earned for prolonged usage.
+    > [!NOTE]
+    > Sustained Use Discounts (SUDs) are handled as after-the-fact adjustments (aka credits) and do not apply to the Discount Handling attribute since this attribute requires that discounts be applied directly to the row they're discounting.
+  - FREE_TIER - free resource usage up to specified limits.
+  - PROMOTION - includes free tial, marketing campaign credits, or other grants to use GCP.
+  - RESELLER_MARGIN - indicates the Reseller Program Discounts earned on every eligible line item.
+  - SUBSCRIPTION_BENEFIT - credits earned by purchasing long-term subscriptions to services in exchange for discounts.
+
+- Where does spot instance usage-based discount fall under? Should there be a **Usage-based discounts** type?
+
+- Where does GCP's sustained usage discount fall under? Maybe the same as spot instance, in which case, **Usage-based discounts**?
```

Comment:
```suggestion
- Where does spot pricing discount fall under? Should there be a **Usage-based discounts** type?
  - We decided to handle spot as PricingCategory = `Dynamic` and PricingSubcategory = `Spot`.
  - We have not yet discussed if there are any special nuances to how spot pricing works that should be explicitly incorporated into Discount Handling beyond identification of spot-priced charges.

- Where does GCP's sustained usage discount fall under? Maybe the same as spot instance, in which case, **Usage-based discounts**?
  - Since GCP SUDs are an after-the-fact discount that includes a negative charge, they're being tracked as ChargeCategory = `Adjustment` and ChargeSubcategory = `Credit`.
  - In the main spec content, SUDs are covered by the "Any price or cost reductions that are awarded after the fact are identified as a `Credit` Charge Subcategory" sentence.
```